templates/index.html

```diff
@@ -65,7 +65,7 @@
 <ul class="nav navbar-top-links navbar-right">
         <!-- /.dropdown -->
 <li class="dropdown">
-    <a class="dropdown-toggle" data-toggle="dropdown" id="user-toggle" aria-haspopup="true" role="button" aria-expanded="false">
+    <a target="_self" href="#" class="dropdown-toggle" data-toggle="dropdown" id="user-toggle" aria-haspopup="true" role="button" aria-expanded="false">
```

Comment:
As a front-end guy, I would suggest using a `<span role="button">` or even a `<button>` tag. CSS wil lthen handle the pointer (` [role="button] {cursor: pointer;}`).

Using `href="#"` could result in odd scroll behavior.

Reply:
Good point. Could you propose a PR to improve it ?